src/tests/ftest/util/dfuse_utils.py

```diff
@@ -94,11 +94,11 @@ def __init__(self, hosts, tmp, dfuse_env=False, log_file=None):
         self.tmp = tmp
         self.dfuse_env = dfuse_env
         self.log_file = log_file
+        self.running_hosts = NodeSet()
 
     def __del__(self):
-        """Destroy Dfuse object and stop dfuse """
-        # stop dfuse
-        self.stop()
+        if len(self.running_hosts):
```

Comment:
We do throw an exception in stop() method if len(self.running_hosts). Do we still need a log in __del__ ?

Reply:
The log in del shouldn't ever happen, it's a safety check as much as anything because as we've seen there's no way for del to actually fail the test.

Reply:
Sure.